specs/cloudbeat.spec.yml

```diff
@@ -25,3 +25,9 @@ inputs:
         - "setup.template.enabled=false"
         - "-E"
         - "gc_percent=${CLOUDBEAT_GOGC:100}"
+  - name: cloudbeat/cis_k8s
+    description: "CIS Kubernetes monitoring"
+    platforms: *platforms

```

Comment:
You need to create anchors for the sections of the first input you are referencing here. For an example see https://github.com/elastic/elastic-agent/blob/06020918dee32333f82b5b23782d450e851bcb29/specs/filebeat.spec.yml#L5-L19

You need `platforms: &platforms` not just `platforms:` in the first definition.

Reply:
Ahhh, that's how that works.